query-engine/connector-test-kit/src/test/scala/writes/ids/IntIdUpdateSpec.scala

```diff
@@ -120,7 +120,7 @@ class IntIdUpdateSpec extends FlatSpec with Matchers with ApiSpecBase {
     result.pathAsLong("data.updateTodo.id") should equal(1)
   }
 
-  "Updating the id of an item with an id field of type Int with autoincrement" should "error" in {
+  "Updating the id of an item with an id field of type Int with autoincrement" should "diagnostics" in {
```

Comment:
Those files look like search and replace errors :-D

Reply:
yes, a little overaggressive find and replace here :D